README.md

```diff
@@ -1,9 +1,10 @@
 # JUSTaMudaeBot
 
-This is a bot that utilizes some aspects of discord.py-self and discum to create a discord self bot that can claim wishes and characters above a min kakera, last minute claim and react to kakera. 
+This is a bot that utilizes some aspects of discord.py-self and discum to create a discord self bot that can claim wishes and characters above a min kakera, last minute claim and react to kakera.
 
 ## Features
-- Can claim characters above a certain kakera 
+
```

Comment:
space?

Reply:
Yes; to be consistent with formatting in the rest of the file.